packages/backend/src/open_payments/grant/service.test.ts

```diff
@@ -133,4 +138,42 @@ describe('Grant Service', (): void => {
       ).resolves.toBeUndefined()
     })
   })
+
+  describe('Update Grant', (): void => {
+    let options: CreateOptions
+    let grant: Grant
+    let authServerId: string
+    beforeEach(async (): Promise<void> => {
+      options = {
+        authServer: faker.internet.url(),
+        accessType: AccessType.IncomingPayment,
+        accessActions: [AccessAction.ReadAll],
+        accessToken: uuid(),
+        managementId: 'gt5hy6ju7ki8',
+        expiresIn: 3000
+      }
+      grant = await grantService.create(options)
+      const authServerService = await deps.use('authServerService')
+      authServerId = (await authServerService.getOrCreate(options.authServer))
+        .id
+    })
+
+    test('can update grant', async (): Promise<void> => {
+      const updateOptions = {
+        accessToken: uuid(),
+        managementId: 'gt5hy6ju7ki8',
+        expiresIn: 3000
+      }
```

Comment:
Can you add a test with `expiresIn: undefined` to confirm that the existing value is still there vs being set to `null` (or should it be set to `null`, maybe if it's `null` in `updateOptions`)?

Reply:
This service really updates the token values, not the grant details. So after rotating a token, if the return value has an `expiresIn` of `null`, I think the`expiresAt` should be set to `undefined` meaning that it does not expire, right?

I'll add a test for that.

Reply:
But for some reason, it does not set `expiresAt` to `undefined`, even when I set `expiresIn` to `undefined` 🤔. I checked the `BaseModel` but we don't set defaults anywhere, do we?